presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,458 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+                                                 become stale.
+
+``resource-groups.exact-match-selector-enabled`` Setting this flag enables usage of an additional            ``false``
+``exact_match_source_selectors`` table to configure
+resource group selection rules defined exact name based
+matches for source, environment and query type. By
+default, the rules are only loaded from the ``selectors``
+table, with a regex-based filter for ``source``, among
+other filters.
+================================================ =========================================================== ===============
 
 Resource Group Properties
 -------------------------
+Resource Groups are defined using a set of properties that determine how resources are allocated and used.
+Here are the key properties that can be set for a Resource Group
+
+* ``name`` (required): The name of the resource group. This is a mandatory property.
 
-* ``name`` (required): name of the group. May be a template (see below).
+* ``maxQueued`` (required): The maximum number of queries that can be queued in the resource group.
+  If this limit is reached, new queries will be rejected.
 
-* ``maxQueued`` (required): maximum number of queued queries. Once this limit is reached
-  new queries will be rejected.
+* ``hardCpuLimit`` (optional): maximum amount of CPU time this
+  group may use in a period.
+
+* ``softMemoryLimit`` (required): The maximum amount of memory that the resource group can use.
+  This can be specified in absolute terms (like "10GB") or as a percentage of the total available
+  memory (like "50%").
 
-* ``hardConcurrencyLimit`` (required): maximum number of running queries.
+* ``hardConcurrencyLimit`` (required): The maximum number of queries that the resource group
+  can run concurrently.
 
-* ``softMemoryLimit`` (required): maximum amount of distributed memory this
-  group may use before new queries become queued. May be specified as
-  an absolute value (i.e. ``1GB``) or as a percentage (i.e. ``10%``) of the cluster's memory.
+* ``softConcurrencyLimit`` (optional): The soft limit on the number of concurrent queries.
+  If this limit is exceeded, the scheduler will try to prevent new queries from starting,
+  but it won't force running queries to stop.
 
 * ``softCpuLimit`` (optional): maximum amount of CPU time this
-  group may use in a period (see ``cpuQuotaPeriod``) before a penalty will be applied to
+  group may use in a period (see ``cpuQuotaPeriod``), before a penalty is applied to
   the maximum number of running queries. ``hardCpuLimit`` must also be specified.
 
-* ``hardCpuLimit`` (optional): maximum amount of CPU time this
-  group may use in a period.
-
-* ``schedulingPolicy`` (optional): specifies how queued queries are selected to run,
-  and how sub-groups become eligible to start their queries. May be one of three values:
+* ``schedulingPolicy`` (optional): The policy that determines how queries are scheduled within the resource group.
```

Comment:
If you un-indent one level for the four sub-items starting with _fair_, you can remove the codeblock formatting line. 

See the difference in the two screenshots. 
**Note** unindenting in the second affect line break and vertical spacing, which I didn't change, so please adjust the four entries to show the line breaks and vertical spacing you have here. 

current code
<img width="828" alt="Screenshot 2023-09-05 at 3 08 23 PM" src="https://github.com/prestodb/presto/assets/7013443/fae39ea7-d7da-404a-9f3f-1b739cd14d02">

unindenting the four sub-items one (and only one) to the left
<img width="826" alt="Screenshot 2023-09-05 at 3 09 35 PM" src="https://github.com/prestodb/presto/assets/7013443/aa141696-6cc7-4ae0-bdb6-9bdfa6378aac">

Reply:
If you addressed this comment, I'm not seeing the change yet. Building the docs locally from the current file still shows the vertical line in the first screenshot.